docs/developer/products.mdx

```diff
@@ -392,7 +392,70 @@ Result:
 The tax equals 1.15 which is 23% of the net price. Gross price is the sum of net price and the tax.
 
 :::note
-Note: The `address` parameter is optional. If it's not provided, Saleor will use the default country code configured in the backend settings (see [DEFAULT_COUNTRY](running-saleor/configuration.mdx#default_country)).
+The `address` parameter is optional. If it's not provided, Saleor will use the default country code configured in the backend settings (see [DEFAULT_COUNTRY](running-saleor/configuration.mdx#default_country)).
+:::
+
+## Stock availability
+
+Stock availability of products is defined for each variant and warehouse (see [Warehouses](/dashboard/configuration/warehouses.mdx) and [Checking inventory](/dashboard/catalog/products.md#checking-inventory) sections to see how to configure it in the Dashboard). There are two fields that allows you to fetch the stock information in the public API.
+
+To fetch the stock availability, use the `Product.isAvailable` field:
```

Comment:
https://docs.saleor.io/docs/next/developer/api-reference/objects/product#isavailable-boolean
Please update API reference (instruction in the readme).

Reply:
Update API reference about stock related changes.